about.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Newton Capstone Project - About Page</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" />
+  <link rel="stylesheet" href="./style.css">
+  <link rel="stylesheet" href="./about.css">
+</head>
+<body>
+  <main>
+    <header>
+      <nav>
+        <div class="nav-socials">
+          <img src="./images/facebook.png" alt="">
+          <img src="./images/twitter.png" alt="">
+          <a href="#">English</a>
+          <a href="#">My page</a>
+          <a href="#">Logout</a>
+        </div>
+        <div class="navbar">
+          <div class="navbar-logo">
+            <label>
+              <a href="./index.html">
+                DEVSPARK
+              </a>
+            </label>
+          </div>
+          <div class="navbar-mobile-close">
+            <img src="./images/Icon-Cancel.svg" alt="">
+          </div>
+          <div class="navbar-menus">
+            <ul>
+              <li>
+                <a href="./index.html">Home</a>
+              </li>
+              <li>
+                <a href="#">Program</a>
+              </li>
+              <li>
+                <a href="#">Join</a>
+              </li>
+              <li>
+                <a href="#">Sponsor</a>
+              </li>
+              <li>
+                <a href="#">News</a>
+              </li>
+              <button>CC Campaign</button>
+            </ul>
+          </div>
+          <div class="navbar-mobile-open">
+            <img src="./images/Icon - Menu.svg" alt="">
+          </div>
+        </div>
+      </nav>
+    </header>
+
+    <section class="about-home">
+      <div class="about-home-text">
+        <h2>"Be among the Best Developers"</h2>
+        <h1>Get Good at Building Websites</h1>
+        <p class="about-home-details">Lorem ipsum dolor sit amet consectetur adipisicing elit. Voluptatem enim obcaecati praesentium veniam nisi suscipit error, modi quisquam eaque fugit asperiores at itaque amet blanditiis mollitia pariatur?</p>
+        <div class="date-time">
+          <h3>Please contact us per Email for any further questions about DEVSPARK</h3>
+          <p><a href="mailto:newtonombese1@gmail.com">info@devspark.com</a></p>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Please make sure your about page headline section elements have the same `font size`, `color`, `height`, `width`, and spacing `padding/spacing` as defined in the original design. For example, your headline title needs to be `uppercase`. Please note that is very important to stick to the original design as much as possible :+1: 

Check the screenshots below to compare and see the differences :point_down: 

####  Original design
![Screenshot from 2022-10-28 18-44-06](https://user-images.githubusercontent.com/6615394/198710820-dbee4473-7cbe-49b6-bc50-ab59c70c4a8b.jpg)

#### Your design
![about](https://github.com/NewtonOmbese/Newton-Capstone-Project/assets/6615394/400ddfef-b9f7-4b85-aa00-95a03894c3a6)

Reply:
Fixed